docker-compose.yml

```diff
@@ -1,8 +1,10 @@
 db:
   build: db
+  image: mattermost/mattermost-prod-db
   volumes:
     - ./volumes/db/var/lib/postgresql/data:/var/lib/postgresql/data
-    - /etc/localtime:/etc/localtime:ro
+  environment:
+    - POSTGRES_PASSWORD
```

Comment:
This is also a breaking change, because people has to setup default value in their environment variable when they run compose.